### ISSUE TITLE
Auto-Complete don't work

### ISSUE THREAD

**Issue by EQuimper** [nkt/atom-autocomplete-modules#28]
Hey, I have this packages for like 3 months and I don't know if the new version of Atom make it don't work but now I lose the autocomplete. I try to reinstall or restart Atom nothing happen. I didn't touch nothing just update Atom to 1.7.4.

**Comment by nkt** [comment]
Hmm, every day I also using this plugin. I'll check it as soon as possible.

**Comment by viczam** [comment]
yep, after the last update only the `require` calls will display the autocomplete menu

**Comment by insraq** [comment]
I can confirm that `import` autocomplete does not work.

**Comment by nkt** [comment]
As I see it's broken by 'babel-language' plugin.

**Comment by ajmichaels** [comment]
Also having the same issue with language-babel

**Comment by iMasterAle** [comment]
Me too.

**Comment by viczam** [comment]
I'm using this one while imports are broken - https://atom.io/packages/autocomplete-js-import

**Comment by AveVlad** [comment]
+1

**Comment by phyllisstein** [comment]
Looks like Babel added a capture for `punctuation.definition.string.begin` to the opening and closing quotation marks in JS string literals, and when the caret is positioned just before the closing quotation mark `autocomplete-plus` uses that as the current scope. My hacky workaround was to change the selector in `completion-provider.js`:

```js
this.selector = '.source.js .string.quoted, .source.js .punctuation.definition.string.begin, .source.coffee .string.quoted';
```

PR, ho!

**Comment by nkt** [comment]
Fixed by @phyllisstein. Published as **1.5.0**.

**Comment by EQuimper** [comment]
Good All work thank ;)
